packages/calling-component-bindings/src/callControlSelectors.ts

```diff
@@ -41,11 +41,25 @@ export type MicrophoneButtonSelector = (
  * @public
  */
 export const microphoneButtonSelector: MicrophoneButtonSelector = reselect.createSelector(
-  [getCallExists, getIsMuted, getDeviceManager, /* @conditional-compile-remove(capabilities) */ getCapabilites],
-  (callExists, isMuted, deviceManager, /* @conditional-compile-remove(capabilities) */ capabilities) => {
+  [
+    getCallExists,
+    getIsMuted,
+    getDeviceManager,
+    /* @conditional-compile-remove(capabilities) */ getCapabilites,
+    /* @conditional-compile-remove(capabilities) */ getRole
+  ],
+  (
+    callExists,
+    isMuted,
+    deviceManager,
+    /* @conditional-compile-remove(capabilities) */ capabilities,
+    /* @conditional-compile-remove(capabilities) */ role
```

Comment:
should this be CC'd with Rooms?

Reply:
It should be for capabilities since role is provided in Teams interop scenario. Also when I stabilize the capabilities feature, I want this code to be stabilized too (given that it is still needed when calling sdk fixes this interop bug)